axum/src/lib.rs

```diff
@@ -444,7 +440,7 @@ pub mod handler;
 pub mod middleware;
 pub mod response;
 pub mod routing;
-#[cfg(feature = "tokio")]
+#[cfg(all(feature = "tokio", feature = "http1"))]
 pub mod serve;
```

Comment:
http2-only is not supported?

Reply:
Oh yes good catch! Change it to `#[cfg(all(feature = "tokio", any(feature = "http1", feature = "http2")))]`